lib/Support/InstanceGraph.cpp

```diff
@@ -249,6 +248,9 @@ InstancePathCache::getAbsolutePaths(ModuleOpInterface op) {
         extendedPaths.push_back(appendInstance(
             path, cast<InstanceOpInterface>(*inst->getInstance())));
       }
+    } else {
+      static InstancePath empty{ArrayRef<InstanceOpInterface>()};
```

Comment:
Is this different than the `static InstancePath empty` from above? If not, consider re-using.  Might make sense for it to live in a single place, for simplicity, although given its implementation pretty sure it'll get thrown into BSS.